File: DigitalOOH.API/DigitalOOH.API/Models/Application/CampaignsModel.cs
namespace DigitalOOH.API.Models.Application
{
    public class CampaignsModel
    {
        public Guid Id { get; set; }
        public required string Name { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        public required string Screens { get; set; }
        public required string Ads { get; set; }
        public DateTime CreatedAt { get; set; }
    }
    public class CampaignsCreateParam
    {
        public required string Name { get; set; }
        public required DateTime StartTime { get; set; }
        public required DateTime EndTime { get; set; }
        public required List<Guid> Screens { get; set; }
        public required List<AdPlaylistItem> Ads { get; set; }
    }

}

File: DigitalOOH.Web/src/app/core/models/campaigns.model.ts
import { adPlaylistItem } from "./ads.model";
import { screenId } from "./screens.model";

export interface campaignsModel {
    id: string;
    name: string;
    startTime: Date;
    endTime: Date;
    screens: string;
    ads: string;
    createdAt: Date;
}

export interface campaignCreateParam {
    name: string;
    startTime: Date;
    endTime: Date;
    screens: screenId[];
    ads: adPlaylistItem[];
}